src/hotspot/share/jvmci/jvmci_globals.cpp

```diff
@@ -204,7 +204,7 @@ void JVMCIGlobals::check_jvmci_supported_gc() {
   if (EnableJVMCI) {
     // Check if selected GC is supported by JVMCI and Java compiler
     if (!(UseSerialGC || UseParallelGC || UseG1GC)) {
-      vm_exit_during_initialization("JVMCI Compiler does not support selected GC", GCConfig::hs_err_name());
+      log_warning(gc)("Selected GC does not support JVMCI: %s", GCConfig::hs_err_name());
```

Comment:
I would add new UL tag `jvmci` if we don't have it yet and use it here. Also message should say `EnableJVMCI is turned off because ...'